server/editor.go

```diff
@@ -122,6 +122,10 @@ func setupEditor(server *Server) {
 				if existingSchema != nil {
 					return fmt.Errorf("ID has been taken")
 				}
+				_, err := schema.NewSchemaFromObj(context["resource"].(map[string]interface{}))
```

Comment:
I think that `context["resource"].(map[string]interface{})` should be extracted to variable.